src/consensus/pbft/pbft.h

```diff
@@ -580,29 +583,31 @@ namespace pbft
     }
 
     void add_configuration(
-      SeqNo seqno,
-      const std::unordered_set<kv::NodeId>& config,
-      const NodeConf& node_conf) override
+      SeqNo seqno, const Configuration::Nodes& config) override
     {
-      if (node_conf.node_id == local_id)
+      auto new_node_id = config.begin()->first;
```

Comment:
config is never empty?

Reply:
PBFT reconfiguration is currently not fully handled and the hook that calls into this only passes one node. I've added a check there and we can revisit when PBFT reconfiguration is addressed.